src/zope/sqlalchemy/tests.py

```diff
@@ -127,15 +131,32 @@ def connect(dbapi_connection, connection_record):
     sa.ForeignKeyConstraint(("user_id",), ("test_users.id",)),
 )
 
-bound_metadata1 = sa.MetaData(engine)
-bound_metadata2 = sa.MetaData(engine2)
-
-test_one = sa.Table(
-    "test_one", bound_metadata1, sa.Column("id", sa.Integer, primary_key=True)
-)
-test_two = sa.Table(
-    "test_two", bound_metadata2, sa.Column("id", sa.Integer, primary_key=True)
-)
+if SA_GE_20:
+    # bound metadata does no longer exist in SQLAlchemy 2.0
+    test_one = sa.Table(
+        "test_one",
+        metadata,
+        sa.Column("id", sa.Integer, primary_key=True)
+    )
+    test_two = sa.Table(
+        "test_two",
+        metadata,
+        sa.Column("id", sa.Integer, primary_key=True)
+    )
+else:
+    bound_metadata1 = sa.MetaData(engine)
+    bound_metadata2 = sa.MetaData(engine2)
```

Comment:
Is `engine2` only used here? If so, it should also be defined in this block.

Reply:
That seems sensible. I moved the definition to here.